assets/js/blocks/reviews/all-reviews/index.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { registerBlockType } from '@wordpress/blocks';
+import Gridicon from 'gridicons';
+
+/**
+ * Internal dependencies
+ */
+import '../editor.scss';
+import Editor from './edit';
+import sharedAttributes from '../attributes';
+import save from '../save.js';
+
+/**
+ * Register and run the "All Reviews" block.
+ */
+registerBlockType( 'woocommerce/all-reviews', {
+	title: __( 'All Reviews', 'woo-gutenberg-products-block' ),
+	icon: {
+		src: <Gridicon icon="grid" />,
+		foreground: '#96588a',
+	},
+	category: 'woocommerce',
+	keywords: [ __( 'WooCommerce', 'woo-gutenberg-products-block' ) ],
+	description: __(
+		'Show product reviews from all products and categories.',
```

Comment:
`Shows a list of all product reviews.`?

Reviews are only for products, not categories. And the name already says they are for products :)